source/enumerate-databases.rst

```diff
@@ -149,6 +174,8 @@ MongoDatabase through MongoClient (e.g. read preference, write concern).
 Drivers SHOULD specify the ``nameOnly`` option when executing the
 ``listDatabases`` command for this method.
 
+Drivers SHOULD implement the ``filter`` option when implementing this method. 
```

Comment:
What do you think about "SHOULD support" vs. "SHOULD implement"?

Reply:
SHOULD support sounds better.